blog/exporters.md

```diff
@@ -74,48 +78,102 @@ The `secadvisor.yml` is used as a parameter when running secadvisor from the com
 The default `secadvisor.yml` has the following fields:
 
 ```
+# host_id is used to reference the agent, should be unique or empty
```

Comment:
host_id is used to identify the webscoket client application, should be unique or empty

Reply:
OK